docs/index.rst

```diff
@@ -1,18 +1,76 @@
 .. aioriak documentation master file, created by
-   sphinx-quickstart on Sat Apr  2 10:53:29 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+    sphinx-quickstart on Sat Apr  2 10:53:29 2016.
+    You can adapt this file completely to your liking, but it should at least
+    contain the root `toctree` directive.
 
 Welcome to aioriak's documentation!
 ===================================
 
-Contents:
+Asyncio (:pep:`3156`) Riak client library.
+This library base on official Basho python client library 
```

Comment:
"This library is based on official Basho python client one". Or even "This project is based on official Bash python client library".